### DISCUSSION TITLE
Alice'in Eşitliği Bulmasına Yardım Edin

### DISCUSSION THREAD

**Discussion by adasue** [sonsuzus/sonsuzus#371] [Soru Cevap]
**Problem:**
Alice’in 2 tane L ve R tam sayıları var. X’in numara olmasına izin verdiğimizde dizideki bütün rakamları **dışlayarak** [L, R] alıyoruz. Alice çok yakın bir zamanda tek ve çift rakamları öğrenmiş ve X’in üzerinde eşitliğin ne olacağını merak ediyor.

**_Giriş Formatı:_**
İlk kısım 2 tane tam sayı içerir L ve R
**_Çıkış Formatı:_**
X tekse “tek” çiftse”çift yazdırın alıntılar olmadan.

_Kısıtlamalar:_
1<=L<=R<= 10¹⁸

**Örnek Girdi**                  
     3      5         

**Örnek Çıktı**
      çift    
                       
_!Dışlama derken özelveya işlevi demek oluyor diğer adıya XOR(Exclusive or) ve sağlanan bağımsız değişkenlerden tek sayıda bağımsız değişken mantıksal olarak doğruysa DOĞRU, aksi takdirde YANLIŞ sonucunu döndürür._

Kaynakça: https://www.hackerearth.com/problem/algorithm/help-alice-to-find-the-parity-f65a1d30/?source=list_view

**Comment by sonsuzus** [comment]
Örnek girdi çıktı var mı
